doc/source/ct.rst

```diff
@@ -2,12 +2,12 @@ Charge Transport
 *************************************
 
 Obtain mobility by Green-Kubo formula
-===============================
+=====================================
 .. automodule:: renormalizer.transport.kubo
    :members:
 
 Charge diffusion simulation dynamics
```

Comment:
The title should be changed to `Charge diffusion dynamics simulation`. It's my fault...